packages/sdk/src/_tests/LinearSdk.test.ts

```diff
@@ -32,11 +32,11 @@ describe("LinearSdk", () => {
 
   it("parses JSON", async () => {
     const sdk = new LinearSdk(
-      resolveWithData({ projectUpdate: { id: "test", diff: JSON.stringify({ some: { nested: { data: 123 } } }) } })
+      resolveWithData({ template: { id: "test", templateData: JSON.stringify({ some: { nested: { data: 123 } } }) } })
     );
-    const response = await sdk.projectUpdate("test");
+    const response = await sdk.template("test");
```

Comment:
This is a change related to the schema update. `ProjectUpdate`.`diff` is now a `JSONObject` rather than `JSON` field. We now use `Template`.`templateData` for the test which is the last entity to use a `JSON` type field on a non-internal field.